pkg/operation/botanist/component/kubeapiserver/deployment.go

```diff
@@ -603,6 +604,16 @@ func (k *kubeAPIServer) admissionPluginNames() []string {
 	return out
 }
 
+func (k *kubeAPIServer) disabledAdmissionPluginNames() []string {
+	var out []string
+
+	for _, plugin := range k.values.DisabledAdmissionPlugins {
```

Comment:
Is it somehow possible to put some validation that only valid plugins can be specified in `admissionPlugins` field of shoot or is such validation already present? Because if we don't do so, users can put any random plugin name and this will lead to kube-apiserver in failed state.

Reply:
Currently, we don't have any validation for this. If we have to introduce it, we have to do it version-wise. I think why we currently don't validate is because these admission plugins in the shoot spec are not that often specified by the users and ideally they shouldn't be putting random plugin names which are not supported by their cluster Kubernetes version for their own safety.

Reply:
Generally, what @acumino is suggesting makes sense and is similar to #3987. Though, we should not do it as part of this PR. Can you please open a dedicated issue for this improvement?

Reply:
Opened https://github.com/gardener/gardener/issues/6559